examples/type-assertion/type_assertion.bal

```diff
@@ -12,13 +12,15 @@ type Employee record {
 };
 
 function assertTypes(anydata emp) {
-    // The `emp` is asserted to be of type `Employee` and if successful the value is assigned to variable `employee`.
+    // The `emp` variable is asserted to be of type `Employee`, and if successful the value is assigned to the variable
+    // `employee`.
     Employee employee = <Employee>emp;
     io:println("Type asserted employee name: ", employee.name);
 
-    // This will panic since emp is inherently an Employee. `trap` is used to handle the error.
+    // Asserting `emp` to be of type `Person` will panic, since `emp` is inherently an `Employee`. `trap` is used to
```

Comment:
Not sure if we can use panic in this context. @pubudu91 WDYT?

Reply:
Panic is the right word here.. But not sure about how it's worded. "will panic" sounds odd to me. Will "will cause a panic" or "will result in a panic" be better?

Reply:
Tried to check how other programming language documentation deal with this. From what I read, I think both the options you suggested sound fine but I prefer the latter option.

@MaryamZi shall we go with "will result in a panic" here?

Reply:
Fixed with 247ac10, thanks!